python/vyos/ifconfig/interface.py

```diff
@@ -1044,21 +1045,59 @@ def apply_mirror(self):
         # Setting up packet mirroring
         ingress_mirror = dict_search('mirror.ingress', self._config)
         if ingress_mirror:
```

Comment:
Why is this `if` required when the next line is `if ingress_mirror in interfaces():`?